### PR TITLE
:sparkles: Homepage Changes

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

- Homepage Updates
- global changes of `ready to code` to `ready-to-code`
- ✅ Launch Workspaces feature flag set to 80% True to 20% False. i.e., 80% of users will see it, & 20% users won't. - from ConfigCat Dashboard

## Related Issue(s)

Fixes #3660

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3664"></a>

### CODE REVIEW

**Review by chrifro — COMMENTED** [review]
Thanks @Siddhant-K-code 🙂

A few comments:

- [ ] the logo animation doesn't include the new logos yet

- [ ] the big logo card e.g. on /security also doesn't includes the new logos yet
- [x] the CTAs here are not in sync with the mockup
Bildschirmfoto 2023-05-22 um 15 46 58

- [x] this headline needs to be updated
Bildschirmfoto 2023-05-22 um 15 49 53

- [x] update this section (headline, reduced content) [see mockup](https://www.figma.com/file/BwA7cNkepuEqD9b39u3VQK/changes-v1?type=design&node-id=402%3A11500&t=lHSgxbHY3uU5aXnd-1)
Bildschirmfoto 2023-05-22 um 15 54 32

- [ ] the newsletter footnote needs to be adjusted (see [figma](https://www.figma.com/file/BwA7cNkepuEqD9b39u3VQK/changes-v1?type=design&node-id=402%3A11652&t=lHSgxbHY3uU5aXnd-1))

File: src/lib/contents/home/features.ts [hunk 1]
```diff
@@ -29,6 +29,10 @@ export const worksOnAnyMachineFeature: Feature = {
   paragraph:
     "No more compatibility issues. Developers can work with the same tools, libraries, and dependencies, regardless of operating system, with a single click.",
   previewComponent: Workspaces,
+  moreButton: {
+    text: "Improve collaboration",
```
[the remark this answers is not in the document]
Comment (Siddhant-K-code) [reply src/lib/contents/home/features.ts:33]: There was a comment in Figma from Talia! I marked it resolved now.

**Comment by Siddhant-K-code** [comment]
> - [ ]  the logo animation doesn't include the new logos yet

this is still being discussed [here](https://www.figma.com/file/BwA7cNkepuEqD9b39u3VQK?node-id=402:11652#445454957) (_internal_), so left untouched. We can easily fix it in followup PR.

File: src/lib/components/index/section-ides.svelte [hunk 2]
```diff
@@ -67,14 +67,19 @@
...
   </p>
-  <div class="text-left sm:text-center m-auto mb-x-small">
-    <LinkButton variant="gray" size="large" href="/cde"
+  <div
+    class="text-left md:text-center m-auto mb-x-small space-x-0 md:space-x-2 space-y-2 md:space-y-0 mt-0 md:mt-8"
+  >
+    <LinkButton variant="primary" size="large" href="/cde"
       >Learn more about CDEs</LinkButton
     >
+    <LinkButton variant="gray" size="large" href="https://gitpod.new"
```
Comment (chrifro) [inline src/lib/components/index/section-ides.svelte:81]: How come you decided to link to /new? We need to be consistent with our buttons. Usually, we link to /workspaces for "try now" which gives a lot more context to new users. Let's stick to that until we made a conscious decision within the team to change that.


For context, this is the screen new users would see on /new
Bildschirmfoto 2023-05-23 um 09 46 08

This is the screen new users would see on /workspaces
Bildschirmfoto 2023-05-23 um 09 49 03

File: src/lib/components/index/get-started/link-git-repository.svelte [hunk 3]
```diff
@@ -2,10 +2,8 @@
   import LinkButton from "$lib/components/ui-library/link-button";
 </script>
 
-<h3>Get started with any project</h3>
-<p class="max-w-md mx-auto">
-  Select a Git provider to start with an existing project from any Git context.
-</p>
+<!-- <h3>Get started with any project</h3> -->
+<p class="max-w-md mx-auto">Choose an existing project from any Git context</p>
```
Comment (chrifro) [inline src/lib/components/index/get-started/link-git-repository.svelte:6]: I couldn't properly test this section as I only see the git provider section and not the example workspaces. The card looks very much wider than in the mockup though. Please make sure both versions are aligned with the mockup
Bildschirmfoto 2023-05-23 um 09 51 19

**Review by chrifro — APPROVED** [review]
Regarding the logos, the two new logos should be reflected on the website as soon as possible. 
Any other logo changes that were is discussion can be paused for now. Your call if this should be part of this PR or if you open a follow-up issue :)

**Comment by Siddhant-K-code** [comment]
> Regarding the logos, the two new logos should be reflected on the website as soon as possible.

Which logos? What/Where did we decided?

**Options:**
- Replace two logos
- Stay with current

I don't think, we should "add" two new logos to the Animation Logo component. as it would break the Animation flow!

File: src/lib/components/index/get-started/link-git-repository.svelte (hunk 3 above)
Comment (Siddhant-K-code) [reply src/lib/components/index/get-started/link-git-repository.svelte]: Resolved it in [`1973667`](https://github.com/gitpod-io/website/pull/3664/commits/19736673d361d1fc37137533c4642935976752eb)

image

**Pull request merged**